lib/Menu.js

```diff
@@ -24,6 +24,12 @@ module.exports = React.createClass({
     ambManager: React.PropTypes.object.isRequired,
   },
 
+  componentDidUpdate: function() {
+    if (this.context.ambManager.isOpen && this.context.ambManager.options.onOpenMenu) {
+      this.context.ambManager.options.onOpenMenu();
+    }
+  },
```

Comment:
Turns out this can cause an infinite loop if your onOpenMenu function calls setState 👎